x/concentrated-liquidity/types/params.go

```diff
@@ -95,6 +101,7 @@ func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 		paramtypes.NewParamSetPair(KeyDiscountRate, &p.BalancerSharesRewardDiscount, validateBalancerSharesDiscount),
 		paramtypes.NewParamSetPair(KeyAuthorizedUptimes, &p.AuthorizedUptimes, validateAuthorizedUptimes),
 		paramtypes.NewParamSetPair(KeyUnrestrictedPoolCreatorWhitelist, &p.UnrestrictedPoolCreatorWhitelist, osmoutils.ValidateAddressList),
+		paramtypes.NewParamSetPair(KeyHookGasLimit, &p.HookGasLimit, validateHookGasLimit),
```

Comment:
Let's make sure that we set this param in the upgrade handler. I suspect that this is what might be breaking e2e